.rubocop.yml

```diff
@@ -49,7 +49,7 @@ Style/AsciiComments:
 Metrics/BlockNesting:
   Description: 'Avoid excessive block nesting'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#three-is-the-number-thou-shalt-count'
-  Enabled: false
+  Enabled: true
```

Comment:
if they are enabled by default I'd suggest removing them from the .rubocop.yml file entirely. IIRC most cops are enabled by default and rubocop suggests using this file only for "customizations".

You can check here if they are enabled by default https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml

We did the same with the other PRs about rubocop

Reply:
Yep, I checked that. I'm in doubt if to keep (to be more explicit) or remove them (to have a cleaner config). The explicit config could be better if we will want to upgrade the rubocop version (the defaults may change).
But I still would prefer a more concise config too.

Reply:
I see that we still have some enabled cops in the config, which are also enabled by default.
So we'll have to do a cleanup, maybe after finishing working on the rubocop task.
For this pr, I'll probably remove these lines, since we did the same with the previous prs.

Reply:
Cool @lightalloy - this Rubocop cleanup team effort is going very well. I read on #2021 that a couple of more contributors are working at sections of the file/violations :)